lib/workspace.go

```diff
@@ -57,7 +58,7 @@ func LoadData(openControlDir string, certificationPath string) (common.Workspace
 	}(&wg)
 	wg.Wait()
 	var errs []error
-	// errs = append(errs, certificationErr)
+	errs = append(errs, certificationErr)
```

Comment:
Hmm, this seems to be causing the build failure... @jcscottiii Happen to remember why it was commented out?